juan-pablo-hernandez/tarea-3/initializeMyName.js

```diff
@@ -0,0 +1,28 @@
+const name1 = 'Jack Ryan Froy';
+
+function initializeNames(name){
```

Comment:
@juan-hm , use modern approaches for array operations , ex. forEach, map, reduce